lib/engine/game/g_1889.rb

```diff
@@ -1,145 +1,12 @@
-          %w[F9] => 'c=r:30,s:2;p=a:2,b:_0;p=a:3,b:_0;p=a:4,b:_0;p=a:5,b:_0;l=K;u=c:80',
-        },
-        gray: {
-          %w[B3] => 't=r:20;p=a:0,b:_0;p=a:_0,b:5',
-          %w[B7] => 'c=r:40,s:2;p=a:1,b:_0;p=a:3,b:_0;p=a:5,b:_0',
-          %w[G14] => 't=r:20;p=a:3,b:_0;p=a:_0,b:4', # TODO?: reference B3 tile, but with rotation
-          %w[J7] => 'p=a:1,b:5',
-        },
-        red: {
-          %w[F1] => 'o=r:yellow_30|brown_60|diesel_100;p=a:0,b:_0;p=a:1,b:_0',
-          %w[J1] => 'o=r:yellow_20|brown_40|diesel_80;p=a:0,b:_0;p=a:1,b:_0',
-          %w[L7] => 'o=r:yellow_20|brown_40|diesel_80;p=a:1,b:_0;p=a:2,b:_0',
-        }
-      }.freeze
-
-      TILES = {
-        '3' => 2,
-        '5' => 2,
-        '6' => 2,
-        '7' => 2,
-        '8' => 5,
-        '9' => 5,
-        '12' => 1,
-        '13' => 1,
-        '14' => 1,
-        '15' => 3,
-        '16' => 1,
-        '19' => 1,
-        '20' => 1,
-        '23' => 2,
-        '24' => 2,
-        '25' => 1,
-        '26' => 1,
-        '27' => 1,
-        '28' => 1,
-        '29' => 1,
-        '39' => 1,
-        '40' => 1,
-        '41' => 1,
-        '42' => 1,
-        '45' => 1,
-        '46' => 1,
-        '47' => 1,
-        '57' => 2,
-        '58' => 3,
-        '205' => 1,
-        '206' => 1,
-        '437' => 1,
-        '438' => 1,
-        '439' => 1,
-        '440' => 1,
-        '448' => 4,
-        '465' => 1,
-        '466' => 1,
-        '492' => 1,
-        '611' => 2,
-      }.freeze
-
-      LOCATION_NAMES = {
-        'A10' => 'Sukumo',
-        'B11' => 'Nakamura',
-        'B3' => 'Yawatahama',
-        'B7' => 'Uwajima',
-        'C10' => 'Kubokawa',
-        'C4' => 'Ohzu',
-        'E2' => 'Matsuyama',
-        'F1' => 'Imabari',
-        'F3' => 'Saijou',
-        'F9' => 'Kouchi',
-        'G10' => 'Nangoku',
-        'G12' => 'Nahari',
-        'G14' => 'Muroto',
-        'G4' => 'Niihama',
-        'H7' => 'Ikeda',
-        'I12' => 'Muki',
-        'I2' => 'Marugame',
-        'I4' => 'Kotohira',
-        'J1' => 'Sakaide & Okoyama',
-        'J11' => 'Anan',
-        'J5' => 'Ritsurin Kouen',
-        'J9' => 'Komatsujima',
-        'K4' => 'Takamatsu',
-        'K8' => 'Tokushima',
-        'L7' => 'Naruoto & Awaji',
-      }.freeze
-
-      MARKET = [
-        %w[75 80 90 100p 110 125 140 155 175 200 225 255 285 315 350],
-        %w[70 75 80 90p 100 110 125 140 155 175 200 225 255 285 315],
-        %w[65 70 75 80p 90 100 110 125 140 155 175 200],
-        %w[60 65 70 75p 80 90 100 110 125 140],
-        %w[55 60 65 70p 75 80 90 100],
-        %w[50y 55 60 65p 70 75 80],
-        %w[45y 50y 55 60 65 70],
-        %w[40y 45y 50y 55 60],
-        %w[30o 40y 45y 50y],
-        %w[20o 30o 40y 45y],
-        %w[10o 20o 30o 40y],
-      ].freeze
-
-      STARTING_CASH = {
-        2 => 420,
-        3 => 420,
-        4 => 420,
-        5 => 390,
-        6 => 390,
-      }.freeze
+      include ::Engine::Config::Game::G1889
```

Comment:
do you need the ::Engine, it can't just be Config::Game?

Reply:
Sure, I'll fix before anything for real. Since we were already in `Game` I thought you need the full thing to go up stack? I'll test and make sure any reference is shortened in anything final